specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Schema.json

```diff
@@ -0,0 +1,274 @@
+        },
+        "startTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The start time for the search."
+        },
+        "lastUpdated": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time of last update."
+        },
+        "eTag": {
+          "type": "string",
+          "description": "The ETag of the search results."
+        },
+        "sort": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SearchSort"
+          },
+          "description": "How the results are sorted."
+        },
+        "requestTime": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The request time."
+        },
+        "aggregatedValueField": {
+          "type": "string",
+          "description": "The aggregated value field."
+        },
+        "aggregatedGroupingFields": {
+          "type": "string",
+          "description": "The aggregated grouping fields."
+        },
+        "sum": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The sum of all aggregates returned in the result set."
+        },
+        "max": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The max of all aggregates returned in the result set."
+        },
+        "schema": {
+          "$ref": "#/definitions/SearchMetadataSchema",
+          "description": "The schema."
+        }
+      },
+      "description": "Metadata for search results."
+    },
+    "SearchSchemaValue": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the schema."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The display name of the schema."
+        },
+        "type": {
+          "type": "string",
+          "description": "The type."
+        },
+        "indexed": {
+          "type": "boolean",
+          "description": "The boolean that indicates the field is searchable as free text."
+        },
+        "stored": {
+          "type": "boolean",
+          "description": "The boolean that indicates whether or not the field is stored."
+        },
+        "facet": {
+          "type": "boolean",
+          "description": "The boolean that indicates whether or not the field is a facet."
+        },
+        "ownerType": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The array of workflows containing the field."
+        }
+      },
+      "required": [
+        "facet",
+        "stored",
+        "indexed"
+      ],
+      "description": "Value object for schema results."
+    },
+    "SearchGetSchemaResponse": {
+      "properties": {
+        "metadata": {
+          "x-ms-client-name": "Metadata",
+          "$ref": "#/definitions/SearchMetadata",
```

Comment:
Perhaps the descriptions are not clear, but I am a bit confused why an API that is described as _Gets the schema for a given workspace._ returns a response object described as: _The metadata from search results_  Does the API output a schema or a search result, or are those interchangeable terms in the context?

Reply:
You are probably right. This is very old and was not well documented. The whole API and payloads should be rewritten. Let's use it as is and I'm adding a work item to fix it.